src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md

```diff
@@ -50,3 +50,34 @@ define(['jquery'], function($) {
   }
 });
 ```
+
+Also, it is possible to adjust existing error message for some specific form field separately.
+
+This is implemented in the core codebase in scope of the [`Magento_CatlogSearch` module]({{ site.mage2bloburl }}/{{page.guide_version}}/app/code/Magento/CatalogSearch/view/frontend/templates/advanced/form.phtml).
+
```

Comment:
Hi @vasilii-b. Thank you for the great example. Could I ask you to add an extended explanation of what exactly needs to be done in order to adjust the message for a specific field? 

The example below shows how it's done in the core but it does not explain the process explicitly. The question that may appear, do we need to modify the `errorPlacement` for adjusting the message or it's enough to add the corresponding item into the `messages` object? 

Thank you!

Reply:
Hi @rogyar,
Thanks for looking into it.
Answering your question - no, the `errorPlacement` does not affect the modification of the message.
The `errorPlacement` adjusts where the validation message will show up when the validation rule not passed.

I'll update the file accordingly.
Thank you!

Reply:
Awesome, thank you!